ci-operator/step-registry/openshift/microshift/e2e/greenboot/openshift-microshift-e2e-greenboot-commands.sh

```diff
@@ -32,7 +32,7 @@ gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
 gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
 gcloud --quiet config set compute/region "${GOOGLE_COMPUTE_REGION}"
 
-cat << 'EOFEOF' > "${HOME}"/greenboot_check.sh
+cat <<'EOFEOF' >"${HOME}"/greenboot_check.sh
```

Comment:
Is it more readable when the whole path is enclosed by quotes, rather than the env itself?
```suggestion
cat << 'EOFEOF' > "${HOME}/greenboot_check.sh"
```